src/Controller/Admin/DashboardController.php

```diff
@@ -52,6 +52,10 @@ public function configureMenuItems(): iterable
         yield MenuItem::section('Editors');
         yield MenuItem::linkToCrud('IP Product Descriptor', 'fas fa-list-alt', InformationProductTypeDescriptor::class);
         yield MenuItem::linkToCrud('IP Digital Resource Descriptor', 'fas fa-list-alt', DigitalResourceTypeDescriptor::class);
+        yield MenuItem::section('Lists');
+        yield MenuItem::linkToUrl('Information Products', 'fas fa-list-alt', $this->generateUrl('pelagos_app_ui_information_products'));
```

Comment:
Shall we use a different icon for this?

Reply:
I added a different icon for the Create New